keytool/src/main/scala/org/constellation/keytool/KeyStoreUtils.scala

```diff
@@ -57,6 +59,25 @@ object KeyStoreUtils {
         }
       }
 
+  def exportPrivateKeyAsHex[F[_]: Sync](
+    path: String,
+    alias: String,
+    storePassword: Array[Char],
+    keyPassword: Array[Char]
+  ): EitherT[F, Throwable, String] =
+    for {
+      keyPair <- keyPairFromStorePath(path, alias, storePassword, keyPassword)
+      hex = KeyUtils.privateKeyToHex(keyPair.getPrivate)
+      _ <- writer(pathDir(path) + privateKeyHexName)
```

Comment:
Will it overwrite the file if it exists(I think it willl)? Maybe we should prepend/append something e.g. part of public key to make it unique?

Reply:
It will. But same thing happens with generating keystore so we would need to fix it in all the places :).

In general `writer` function has `TODO` inside:
https://github.com/Constellation-Labs/constellation/blob/f59b2e2197b0be4cc6d641e387d62c16259ba896/keytool/src/main/scala/org/constellation/keytool/KeyStoreUtils.scala#L148

Reply:
That's true. I had it implemented for a little while in the comakery-hot-wallet (before it turned out not needed after all ;)). I was just checking if the file exists under that path and if it did I just threw an exception. It was like 3 lines of code and probably good enough for CLI app.

Reply:
Done